### PR TITLE
Fix bug with wrong index of CYCLE argument processing.

### PR DESCRIPTION
**Pull request by pomalley**
The QS is looking at the wrong index for converting the CYCLE command from number of cycles desired to counter index.

### CODE REVIEW

File: QubitServer/src/main/java/org/labrad/qubits/jumptable/JumpTable.java [hunk 1]
```diff
@@ -38,11 +38,11 @@ public void addEntry(String name, Data argument) {
     if (name.equals("CYCLE")) {
       List<Data> args = argument.getClusterAsList();
       Preconditions.checkArgument(args.size() == 4, "Cycle must have 4 arguments; currently has " + args.toString());
-      if (countersUsed == 3) {     // TODO: get num counters from hardware
+      if (countersUsed == 4) {     // TODO: get num counters from hardware
```
Comment (maffoo) [inline QubitServer/src/main/java/org/labrad/qubits/jumptable/JumpTable.java]: can we factor this out into a class-level static final constant for now? The TODO about getting the value from hardware instead can go with the constant.

**Comment by pomalley** [comment]
Did that

**Comment by pomalley** [comment]
Julian says it works.

**Comment by pomalley** [comment]
Also what's the deployment strategy these days?

**Comment by maffoo** [comment]
Please move the TODO comment up to the constant as well.

As for deployment, you'll want to bump the version on the server, then just run `sbt packArchive` to produce a `.tar.gz` file in the `target` directory of the project. Unpack that on the server under `C:\labrad\servers` and delete the version that was there. Refresh the node and you should see it pick up the new version. Automation of all this coming soon...

**Comment by pomalley** [comment]
OK. Just wanted to make sure we're no longer committing binaries.

Found another bug to do with mutable data, and added an offset to deal with JT entry index counting on the fpga_server side of things. This has now been tested to work on Vince, and Julian should have a pyle PR incoming soon that has fixes on that end.

**Comment by maffoo** [comment]
LGTM

**Comment by pomalley** [comment]
Sweet, thanks.

**Pull request merged**